### PR TITLE
Dashboard responsiveness

### PR DESCRIPTION
**Pull request by vdequercize**
In this update, I
- replaced the homepage journey length placeholder with actual data
- updated the seeds so that each journey would have a different number of days
- made dashboard page responsive
- updated dashboard cards to make them consistent with featured trips
- updated readme page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,9 +23,3 @@ Back-end :
 Integrations :
 
 - Mapbox API
-
-
-Noteable changes that are in development:
-
-- Removal of countries not yet seeded
-- Making website responsive to majority of device sizes
```
Comment (vdequercize) [inline README.md:29]: Updated read me page to give context about the project

File: app/assets/stylesheets/pages/_dashboard.scss [hunk 2]
```diff
@@ -68,11 +73,108 @@
...
+      justify-content:center;
+      align-content:center;
+      height: min-content;
+    }
+  }
+
+  .nav-title {
+    font-size: medium;
+  }
+}
```
Comment (vdequercize) [inline app/assets/stylesheets/pages/_dashboard.scss:180]: Made dashboard page responsive

File: app/views/devise/registrations/new.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <h2 class="mx-auto pt-4 devise-container">Sign up</h2>
 
-<div class="d-flex justify-content-center border w-50 mx-auto">
+<div class="d-flex justify-content-center w-50 mx-auto">
   <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), data: { turbo: :false }) do |f| %>
     <%= f.error_notification %>
```
Comment (vdequercize) [inline app/views/devise/registrations/new.html.erb:6]: Removed border from sign up page

File: app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <h2 class="mx-auto pt-4 devise-container">Log in</h2>
 
-<div class="d-flex justify-content-center border w-50 mx-auto pt-3 pb-3">
+<div class="d-flex justify-content-center w-50 mx-auto pt-3 pb-3">
   <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name), data: { turbo: :false }) do |f| %>
     <div class="form-inputs">
       <%= f.input :email,
```
Comment (vdequercize) [inline app/views/devise/sessions/new.html.erb:6]: Removed border from log in page

File: app/views/pages/dashboard.html.erb [hunk 5]
```diff
@@ -25,11 +25,22 @@
...
+                  <%= "🌊 Sea 🌊" %>
+                  <% when "Major City" %>
+                  <%= "🌉 City 🌉" %>
+                  <% when "Mountain" %>
+                  <%= "🏔 Mountain 🏔" %>
+                  <% end %>
+                </p>
                 <div id="dates">
                   <p> From <%= journey.start_date.strftime("%a %d %B") %> to <%= journey.end_date.strftime("%a %d %B") %></p>
                 </div>
```
Comment (vdequercize) [inline app/views/pages/dashboard.html.erb:46]: Changed tag format and cleaned up the page

File: db/seeds.rb [hunk 6]
```diff
@@ -447,75 +447,75 @@
...
+                        end_date: "09/03/2022",
                         country: "France",
                         user: user_1)
 food_3 = Journey.create!(name: "Feels like home",
                           tag: "Food",
                           start_date: "03/03/2022",
-                          end_date: "08/08/2022",
+                          end_date: "18/03/2022",
                           country: "France",
                           user: user_1)
```
Comment (vdequercize) [inline db/seeds.rb:521]: Updated seeds date

**Review by AJCJ1 — APPROVED** [review]
Changes noted

**Pull request merged**